containers/src/Kokkos_DynamicView.hpp

```diff
@@ -183,7 +355,7 @@ class DynamicView : public Kokkos::ViewTraits<DataType, P...> {
   KOKKOS_INLINE_FUNCTION
   size_t size() const noexcept {
     size_t extent_0 =
-        *reinterpret_cast<const size_t*>(m_chunks + m_chunk_max + 1);
+        *reinterpret_cast<const size_t*>(m_chunks_host + m_chunk_max + 1);
```

Comment:
`m_chunks_host` -> `m_chunks`?

Reply:
that was actually  a request from me adding the host since this is a host side pointer.